src/libra_py/workflows/nbra/lz.py

```diff
@@ -378,6 +378,7 @@ def run(H_vib, params):
                        "Boltz_opt":1, "Boltz_opt_BL":1, "T":300.0,
                        "do_output":True, "outfile":"_out.txt", "do_return":True,
                        "evolve_Markov":True, "evolve_TSH":True, 
+                       "extend_md_option":0, "extend_md_time":0,

```

Comment:
please document these new keywords in the docstring

Reply:
Added, and I also fixed a small bug from copying